packages/wrangler/src/dev/remote.ts

```diff
@@ -114,7 +114,7 @@ export async function createRemoteWorkerInit(props: {
 		props.modules
 	);
 
-	const legacyAssets = await syncLegacyAssets(
+	const legacyAssets = await syncWorkersSite(
```

Comment:
NIT: rename `legacyAssets` to `workersSitesAssets`?

Reply:
2bc289e2e